src/js/menu/menu-button.js

```diff
@@ -37,6 +37,16 @@ class MenuButton extends ClickableComponent {
   }
 
   /**
+   * Hide the menu if the number of items is less than or equal to this threshold. This defaults
```

Comment:
That would avoid the multiple uses of ` if (this.player().tech_ && this.player().tech_.featuresNativeTextTracks) {` in this component - just increment the threshold count at the same time it adds the Captions Settings Menu Item.